src/librustdoc/doctest.rs

```diff
@@ -360,6 +360,9 @@ fn run_test(
     for debugging_option_str in &rustdoc_options.debugging_opts_strs {
         compiler.arg("-Z").arg(&debugging_option_str);
     }
+    for remap_path_prefix_str in &rustdoc_options.remap_path_prefix_strs {
+        compiler.arg("--remap-path-prefix").arg(&remap_path_prefix_str);
```

Comment:
It looks like this is the only place you use remap-path-prefix. I think you should also pass it to the compiler options in `core.rs`, as well as the options used for parsing doctests.